sdcm/cluster.py

```diff
@@ -2803,17 +2803,19 @@ def get_nodetool_status(self, verification_node=None):  # pylint: disable=too-ma
             verification_node = random.choice(self.nodes)
         status = {}
         res = verification_node.run_nodetool('status')
-        data_centers = res.stdout.strip().split("Data center: ")
+        data_centers = res.stdout.strip().split("Datacenter: ")
         for dc in data_centers:
             if dc:
                 lines = dc.splitlines()
                 dc_name = lines[0]
                 status[dc_name] = {}
                 for line in lines[1:]:
+                    if line.startswith('--'):  # ignore the title line in result
+                        continue
                     try:
                         state, ip, load, _, tokens, owns, host_id, rack = line.split()
                         node_info = {'state': state,
-                                     'load': load,
+                                     'load': '%s%s' % (load, _),
```

Comment:
What does it mean??? confusing

Reply:
unit string is assigned to `_` in the past, here I just joined them together.
Original result only contains the number without the unit string.

Reply:
I will rename the `_` to `load_unit`